var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -86,10 +89,12 @@ def install(self, spec, prefix):
         options.append("--with-c=%s" % ("yes" if "+c" in spec else "no"))
         options.append("--with-python=%s" % ("yes" if "+python" in spec else "no"))
         options.append("--with-java=%s" % ("yes" if "+java" in spec else "no"))
-        options.append("--with-go=%s" % ("yes" if "+go" in spec else "no"))
-        options.append("--with-lua=%s" % ("yes" if "+lua" in spec else "no"))
-        options.append("--with-php=%s" % ("yes" if "+php" in spec else "no"))
-        options.append("--with-qt4=%s" % ("yes" if "+qt4" in spec else "no"))
+        options.append("--with-go=no")
+        options.append("--with-lua=no")
+        options.append("--with-php=no")
+        options.append("--with-kotlin=no")
+        options.append("--with-ruby=no")
+        options.append("--with-qt4=no")
```

Comment:
@tgamblin @alfredo-gimenez Any objections to disabling these options (since there appear to be no corresponding variants for them)?

Reply:
Could the variant propagation mechanism detailed [here](https://spack.readthedocs.io/en/latest/basic_usage.html#variants) cause these options to activate, even though said variant isn't explicitly defined in that package?

Reply:
@haampie do you think these changes are safe to add?

Reply:
IMO I think the issue isn't so much if they are safe or not to add but getting the build recipe "right" (e.g., dependencies, versions of dependencies) for those that are added.